docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -13,32 +13,19 @@ weight: 200
 
 # Configure generic OAuth authentication
 
```

Comment:
Would love an intro here about what authentication is, 2 or 3 sentences, with a link out to more information about authentication is. Think about it this way - a user does a google search and lands on this page. What content do you want them to see in order to get oriented?

Reply:
We've got a short intro here: https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/. We could extend it a bit more when we update this page. Do you think that's enough, or should we have a quick into to authentication on all auth provider pages? Alternatively, we could also reference [this page](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/) from the top of the generic auth page.